lib/rex/post/meterpreter/ui/console.rb

```diff
@@ -75,11 +75,12 @@ def interact(&block)
   #
   # Interacts with the supplied channel.
   #
-  def interact_with_channel(channel)
+  def interact_with_channel(channel, raw: false)
     channel.extend(InteractiveChannel) unless (channel.kind_of?(InteractiveChannel) == true)
     channel.on_command_proc = self.on_command_proc if self.on_command_proc
     channel.on_print_proc   = self.on_print_proc if self.on_print_proc
     channel.on_log_proc = method(:log_output) if self.respond_to?(:log_output, true)
+    channel.raw = raw
```

Comment:
I couldn;t come up with a better name than `raw` we already use interact/interactive/interacting all over the place taking suggestions :sweat_smile: